packages/cfd/src/Constants/jurisdiction-svg-contents.ts

```diff
@@ -0,0 +1,82 @@
+import { localize } from '@deriv/translations';
```

Comment:
Lets create a folder jurisdiction-contents and move all these content files as subfolders